docs/plugins/preset-stage-2.md

```diff
@@ -9,6 +9,7 @@ package: babel-preset-stage-2
 This preset includes the following plugins:
 
 - [transform-object-rest-spread](/docs/plugins/transform-object-rest-spread)
+- [transform-decorators](/docs/plugins/transform-decorators)
```

Comment:
Yeah the plugin implementation didn't change so we should keep the message (although yes the spec changed)